lightstep/resource_metric_dashboard.go

```diff
@@ -120,50 +135,50 @@ func resourceMetricDashboardRead(ctx context.Context, d *schema.ResourceData, m
 
 	c := m.(*client.Client)
 
-	dashboard, err := c.GetMetricDashboard(ctx, d.Get("project_name").(string), d.Id())
+	dashboard, err := c.GetUnifiedDashboard(ctx, d.Get("project_name").(string), d.Id())
 	if err != nil {
 		apiErr := err.(client.APIResponseCarrier)
 		if apiErr.GetHTTPResponse() != nil && apiErr.GetHTTPResponse().StatusCode == http.StatusNotFound {
 			d.SetId("")
 			return diags
 		}
-		return diag.FromErr(fmt.Errorf("Failed to get metric dashboard: %v\n", apiErr))
+		return diag.FromErr(fmt.Errorf("failed to get metric dashboard: %v", apiErr))
 	}
 
 	if err := setResourceDataFromMetricDashboard(d.Get("project_name").(string), *dashboard, d); err != nil {
-		return diag.FromErr(fmt.Errorf("Failed to set metric dashboard from API response to terraform state: %v", err))
+		return diag.FromErr(fmt.Errorf("failed to set metric dashboard from API response to terraform state: %v", err))
 	}
 
 	return diags
 }
 
-func getMetricDashboardAttributesFromResource(d *schema.ResourceData) (*client.MetricDashboardAttributes, error) {
+func getUnifiedDashboardAttributesFromResource(d *schema.ResourceData) (*client.UnifiedDashboardAttributes, error) {
```

Comment:
I didn't move this out of `resource_metric_dashboard.go` so it would be clear it has only been renamed, not functionally changed. I can move it in a follow-up.